etc/nginx/ssl/cfssl.json

```diff
@@ -0,0 +1,20 @@
+{
+  "CN": "developer-portal-127-0-0-1.nip.io",
+  "hosts": [
+    "developer-portal-127-0-0-1.nip.io",
+    "demos.developer-portal-127-0-0-1.nip.io"
```

Comment:
@limed Any particular reason to havee this extra subdomain?

Reply:
No particular reason, I copied the domains from the openssl config file we had. If its not needed it can be removed

Reply:
Removed in 3b21e11b8ab2129e8ef41220ffc0b83b45c87d46